Source/GMCAbilitySystem/Private/Components/GMCAbilityComponent.cpp

```diff
@@ -11,6 +11,7 @@
 #include "Attributes/GMCAttributesData.h"
 #include "Effects/GMCAbilityEffect.h"
 #include "Net/UnrealNetwork.h"
+#include "WorldPartition/ContentBundle/ContentBundleLog.h"
```

Comment:
what's this for?

Reply:
Probably something erroneously autocompleting and the IDE thus including the header.